snapcraft/plugins/cmake.py

```diff
@@ -19,6 +19,9 @@
 These are projects that have a CMakeLists.txt that drives the build.
 The plugin requires a CMakeLists.txt in the root of the source tree.
 
+This plugin supports the keywords from the make plugin - check the
+'make' topic for more details.
```

Comment:
I suggest rephrasing just slightly: "This plugin also supports options from the `make` plugin. Run `snapcraft help make` for more details."